bin/disco_ssh.py

```diff
@@ -133,16 +135,31 @@ def build_ssh_cmd(self, ips):
         """
         Given a list of ip addresses, build an ssh command to tunnel through n-1 ips to reach the n-th ip
         """
-        command = " ".join([
-            "ssh -At {} {}".format(SSH_OPTIONS, ip)
-            for ip in ips])
+        if self.tunnel:
+            lport, rport = self.tunnel.split(":")
+            host = self.args["<host>"]
```

Comment:
The problem is that the target is not necessarily an EC2 instance.  We're using it to get to RDS instances.  Possible solutions:

- expand the host search to handle RDS instances too.  I'm not eager to go down this path.
- specify the target with a flag, e.g. --target xyx-txdb.aws.wgen.net.  I personally find this less intuitive than using the host argument, but maybe that's just me.
 - create an entirely new command, disco_tunnel.py, for this purpose.  Another path I'm not eager to go down.
 - leave as is

Which is best?